extensions/amp-video/0.1/amp-video.js

```diff
@@ -618,6 +619,19 @@ class AmpVideo extends AMP.BaseElement {
     return ranges;
   }
 
+  /**
+    * @override
+    **/
+  firstLayoutCompleted() {
+    const placeholder = this.getPlaceholder();
+    if (placeholder && placeholder.classList.contains('i-amphtml-blur') &&
+      isExperimentOn(this.win, 'blurry-placeholder')) {
+      setImportantStyles(placeholder, {'opacity': 0});
```

Comment:
Add a comment explaining that this will trigger a CSS fade out animation for the placeholder.